### ISSUE TITLE
Parsing error: Unexpected token <

### ISSUE THREAD

**Issue by PLQin** [eslint/eslint#11233]
yes , I saw a ` # 9917 `, But my situation is different from his ;

**Tell us about your environment**

* **ESLint Version:** v5.11.1
* **Node Version:** v10.14.2
* **npm Version:** 6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?**

- .eslintrc.js

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    root: true,
    parserOptions: {
        "parser": "babel-eslint", 
        "sourceType": "module",
        "ecmaVersion": 6
    },
    env: {
        browser: true,
        node: true,
        es6: true,
    },
    extends: ["eslint:recommended"],

    // add your custom rules here
    //it is base on https://github.com/vuejs/eslint-config-vue
    rules: {
        "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
        "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off",
        "no-undef": "off",
        "no-unused-vars": "off",
        "no-redeclare": "off",
        "no-empty": "off",
        "indent": ["warn", 4],
        'vue/max-attributes-per-line': 'off', 
        'vue/order-in-components': 'off',
        'vue/html-end-tags': 'off',
    }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

vue code
```
<template>
  <div id="app">
    <router-view></router-view>
  </div>
</template>

<script>
export default {
  name: 'App' 
};
</script>
```

run ESLint : 

```bash
eslint --fix "./src/**/*.{js,vue}"
```

**What did you expect to happen?**

good running 

**What actually happened? Please include the actual, raw output from ESLint.**

```
D:\_Me\-COMPANY-JiTeng-\src\App.vue : 
 1:1  error  Parsing error: Unexpected token <
```

**Are you willing to submit a pull request to fix this bug?**

![image](https://user-images.githubusercontent.com/24371471/50627561-f0115e00-0f6e-11e9-8d67-efed11f368ed.png)

**Comment by g-plane** [comment]
You should use `vue-eslint-parser`, and you can check `eslint-plugin-vue` for details.

**Comment by PLQin** [comment]
`vue-eslint-parser` ?  Using ` eslint-plugin-vue ` should work, I'll try;

**Comment by g-plane** [comment]
You just need to install `eslint-plugin-vue`, then `vue-eslint-parser` will be installed at the same time. Next, change your ESLint config and change the parser from `babel-eslint` to `vue-eslint-parser`. Also don't forget add `parser: 'babel-eslint'` to `parserOptions`.

I recommend you install and run ESLint locally, not globally.

**Comment by plqin-xiaohao** [comment]
> You just need to install `eslint-plugin-vue`, then `vue-eslint-parser` will be installed at the same time. Next, change your ESLint config and change the parser from `babel-eslint` to `vue-eslint-parser`. Also don't forget to add `parser: 'babel-eslint'` to `parserOptions`.
> 
> I recommend you to install and run ESLint locally, not globally.

yoyo, I had an Epiphany , thank, thanks

**Comment by g-plane** [comment]
If your problem is solved, you can close this issue.
